src/components/calcite-input/calcite-input.tsx

```diff
@@ -365,6 +367,9 @@ export class CalciteInput implements LabelableComponent, FormComponent {
   //--------------------------------------------------------------------------
 
   keyDownHandler = (event: KeyboardEvent): void => {
+    if (event.key === "ArrowUp") {
```

Comment:
For readability, could you add a small comment on why this only is needed for the up key?